content/akeneo-event-platform/getting-started.md

```diff
@@ -1,4 +1,154 @@
 # Getting started
 
-::: panel-link <!-- TODO TEXT PANEL LINK -->[Next](/akeneo-event-platform/api-reference.html)
-:::
\ No newline at end of file
+This guide will walk you through the essential steps to set up your environment, create subscriptions, and trigger events.
+
+### Prerequisites
+
+Before proceeding, ensure that you have an active App (it can be a custom App) or an active connection to an Akeneo PIM.
```

Comment:
Is it clear for everyone what an app, a custom app or a connection is? Maybe we should add a link to a page that explains this.

Reply:
links added